src/data/reducers/shopify.js

```diff
@@ -88,6 +88,18 @@ const getProduct = id => {
   };
 };
 
+// Gets all the collections from Shopify
+const getCollections = () => {
+  return dispatch => {
+    client.collection.fetchAllWithProducts().then(res => {
```

Comment:
Alternatively, `fetchByHandle` can be used with the parameter `bestsellers` but I'm not sure how we would store that in our state as well and if we plan to use other collections - this is more reusable